compiler/rustc_target/src/spec/aarch64_linux_android.rs

```diff
@@ -16,6 +16,7 @@ pub fn target() -> Target {
             features: "+neon,+fp-armv8".into(),
             supported_sanitizers: SanitizerSet::CFI
                 | SanitizerSet::HWADDRESS
+                | SanitizerSet::KCFI
```

Comment:
These are userspace targets that don't compile their libc with kcfi, right? I don't think mixing non-kcfi and kcfi object files is possible (and based on the slides I think mixing kcfi object files not generated by the same lto invocation isn't possible either), so this probably shouldn't have kcfi support. Maybe enable it for the -none target specs? Those are meant for bare metal usage like kernels.

Reply:
Indeed. I just changed it to those targets instead.